build.sbt

```diff
@@ -150,7 +150,7 @@ lazy val docSettings = allSettings ++ Seq(
     "-doc-root-content", (resourceDirectory.in(Compile).value / "rootdoc.txt").getAbsolutePath
   ),
   scalacOptions ~= {
-    _.filterNot(Set("-Yno-predef", "-Xlint"))
```

Comment:
It would also work to do this just for `scalacOptions in Tut`, right?

Reply:
I don't think so based on https://github.com/tpolecat/tut/issues/105. But I suppose we could `tutScalacOptions`. Its also odd cause it seems like tut should already be filtering this out based on https://github.com/tpolecat/tut/pull/108.

Reply:
That issue is quite old, though. I actually needed to do this [this morning for circe](https://github.com/circe/circe/commit/6ada85a04ec84125a812c3650c527e46854337c7), and (surprisingly) the `in Tut` fixed the issue.

Reply:
Good to know it works. We could do that instead. I only added there because that's how we'd added `-Xlint` and `-Yno-predef` previously. Happy to put it in the base settings though.